maven-projects/target/.plxarc

```diff
@@ -0,0 +1 @@
+maven-shared-archive-resources
```

Comment:
The `target` directory is generated and don't include to the PR.

Reply:
done